binary_encode_override_test.go

```diff
@@ -0,0 +1,36 @@
+package amino_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tendermint/go-amino"
+)
+
+type Thing struct {
```

Comment:
Can we also add a test case for a byte slice alias?

Reply:
Done.